FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/MuonTaggerPhiTheta.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+
+  <info name="FCCeeMuonTagger"
+        title="Muon Tagger for ALLEGRO, with phi-theta segmentation"
+        author="Created by A. Duglishvili from MuonTagger.xml, moving from eta to theta segmentation"
+        url="no"
+        status="development"
+        version="1.0">
+    <comment>
+      Simple muon tagger with phi-theta segmentation - barrel and endcaps
+    </comment>
+  </info>
+
+  <define>
+    <constant name="MuonTaggerBarrelLayers" value="2"/>
+    <constant name="MuonTaggerEndcapLayers" value="2"/>
+  </define>
+
+  <display>
+    <vis name="muon_vis" r="1.0" g="0.1" b="0.1" showDaughters="true" visible="true" />
+  </display> 
+  
+  <readouts>
+    <readout name="MuonTaggerBarrelPhiTheta">
+      <segmentation type="FCCSWGridPhiTheta_k4geo" phi_bins="704" offset_phi="-pi+(pi/704.)" grid_size_theta="0.010013373" offset_theta="0.10884388"/>
+      <id>system:4,subsystem:1,layer:5,theta:10,phi:10</id>
+    </readout>
+    <readout name="MuonTaggerEndcapPhiTheta">
+      <segmentation type="FCCSWGridPhiTheta_k4geo" phi_bins="704" offset_phi="-pi+(pi/704.)" grid_size_theta="0.010013373" offset_theta="0.10884388"/>
+      <id>system:4,subsystem:1,layer:5,theta:10,phi:10</id>
+    </readout>
+  </readouts>
+
+  <detectors>
+    <!-- barrel -->
+    <detector id="DetID_Muon_Barrel" name="MuonTaggerBarrel" type="SimpleCylinder_o1_v03" sensitive="true" vis="muon_vis" readout="MuonTaggerBarrelPhiTheta">
+      <!-- added for Pandora -->
+      <type_flags type="DetType_CALORIMETER + DetType_MUON + DetType_BARREL"/>
+      <!-- end -->
+      <!-- sensitive type="SimpleTrackerSD"/ -->
+      <sensitive type="BirksLawCalorimeterSD"/>
```

Comment:
A comment about why a muon detector is using a calorimeter SD action may help on the long term